### ISSUE TITLE
something about the paper

### ISSUE THREAD

**Issue by Wi1L-Y** [seemoo-lab/frankenstein#8]
### 4 help
Sorry to be a bother. I read your paper titled as "Frankenstein_ Advanced Wireless Fuzzing to Exploit New Bluetooth Escalation Targets "​ and I am very interested in it.
I have some questions about the paper. Could you provide an email address?

**Comment by bolek42** [comment]
Hi,
I have dropped you an email from my @mailbox.org address. But if the question is not confidential (e.g. about new vulnerabilities) I'd encourage you to start a public discussion, because other people might have a similar question ;)
best, JR

**Comment by jiska2342** [comment]
Since this project is based on reverse engineering, there's not much documentation. We extracted some names of the memory sections and global variable names from Cypress WICED Studio 6.2. We also defined a memory map based on what we understood so far, which is available here: https://github.com/seemoo-lab/internalblue/blob/master/internalblue/fw/fw_0x4208.py#L45

Not sure why `0x20007` is created and how (@bolek42 is more into the whole executable generation process). If you're not missing a 0, it's some address in ROM within the function `DHM_UpdateACLStats`, so it doesn't really make sense...
